lnrpc/routerrpc/router.proto

```diff
@@ -209,6 +209,46 @@ message ChannelUpdate {
 	// satoshi.
 	uint32 fee_rate = 9;
 }
+message ResetMissionControlRequest{}
+
+message ResetMissionControlResponse{}
+
+message QueryMissionControlRequest {}
+
+/// QueryMissionControlResponse contains mission control state per node.
+message QueryMissionControlResponse {
+    repeated NodeHistory nodes = 1;
+}
+
+/// NodeHistory contains the mission control state for a particular node.
+message NodeHistory {
```

Comment:
These protos should set `json_name` so it's consistent with the display of the rest of our fields on the command line.

Reply:
Isn't this only relevant for the rest interface, which we don't have yet?

Reply:
Ok, see what you mean now. But this is not possible anymore because we use the custom encoder now.

Reply:
Added `json_name` anyway, although still not sure who is going to benefit from this.